webapp/src/models/NotificationType.d.ts

```diff
@@ -1,12 +1,12 @@
 type NotificationType =
   | 'IntersectionReferenceAlignmentNotification'
   | 'SignalGroupAlignmentNotification'
-  // | "SpatMinimumDataNotification" //
-  // | "MapMinimumDataNotification" //
+  | 'SpatMinimumDataNotification' //
```

Comment:
Should we clean out the extra comment slashes in here while we're adding these types in?